post-processor/vsphere-template/step_create_folder.go

```diff
@@ -0,0 +1,72 @@
+package vsphere_template
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/packer/packer"
+	"github.com/mitchellh/multistep"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+)
+
+type StepCreateFolder struct {
+	Folder string
+}
+
+func (s *StepCreateFolder) Run(state multistep.StateBag) multistep.StepAction {
+	ui := state.Get("ui").(packer.Ui)
+	ctx := state.Get("context").(context.Context)
+	f := state.Get("finder").(*find.Finder)
+	d := state.Get("datacenter").(string)
+
+	if s.Folder != "" {
+		ui.Say("Creating or checking destination folders...")
+
+		if !strings.HasPrefix(s.Folder, "/") {
+			s.Folder = filepath.Join("/", s.Folder)
+		}
```

Comment:
Yes if all paths must be bound to the root, it's better to require that in `Prepare`. 

It's less obvious that `/some/path` is the same thing as `some/path`.